packages/logger/src/index.ts

```diff
@@ -14,8 +46,92 @@ export const consoleLogger =
     return logger
   }
 
+export const nodeConsoleLogger = Console instanceof Function ? new Console(process.stdout, process.stderr) : console
+
+export class QuietLogger {
+  private isDebug: boolean
+
+  constructor(
+    public name: string,
+    public parallelConsoleLog: boolean = false
+  ) {
+    this.isDebug = debug.enabled(name)
+  }
+
+  debug(message: any, ...optionalParams: any[]): void {
+    if (!this.isDebug) return
+
+    const text = this.formatMessage(message, 'debug')
+    this.callLogMethods('debug', text, ...optionalParams)
+  }
+
+  error(message: any, ...optionalParams: any[]): void {
+    const text = this.formatMessage(message, 'error')
+    this.callLogMethods('error', text, ...optionalParams)
+  }
+
+  info(message: any, ...optionalParams: any[]): void {
+    const text = this.formatMessage(message, 'info')
+    this.callLogMethods('info', text, ...optionalParams)
+  }
+
+  log(message: any, ...optionalParams: any[]): void {
+    if (!this.isDebug) return
+    const text = this.formatMessage(message, 'log')
+    this.callLogMethods('log', text, ...optionalParams)
+  }
+
+  warn(message: any, ...optionalParams: any[]): void {
+    const text = this.formatMessage(message, 'warn')
+    this.callLogMethods('warn', text, ...optionalParams)
+  }
+
+  private callLogMethods(level: string, text: string, ...optionalParams: any[]): void {
+    const colorizedOptionalParams = optionalParams.map((param: any) => this.formatObject(param))
+    // @ts-ignore
+    nodeConsoleLogger[level](text, ...colorizedOptionalParams)
+    if (this.parallelConsoleLog) {
+      // @ts-ignore
+      console[level](text, ...colorizedOptionalParams)
+    }
+  }
+
+  private formatMessage(message: any, level: string): string {
+    const colorizedLevel = colors[level](level.toUpperCase())
+    const colorizedScope = colors['scope'](this.name)
+    const colorizedDate = colors['date'](DateTime.utc().toISO())
+    const colorizedMessageText =
+      typeof message === 'string' ? colors[`${level}_text`](message) : this.formatObject(message)
+    return `${colorizedDate} ${colorizedLevel} ${colorizedScope} ${colorizedMessageText}`
+  }
+
+  private formatObject(param: any): string {
+    if (param instanceof Error) {
+      return colors['object_error'](param.stack || `${param.name}: ${param.message}`)
+    } else if (['string', 'number', 'boolean', 'bigint'].includes(typeof param)) {
+      return colors['object'](param)
+    }
+    return colors['object'](JSON.stringify(param, null, 2))
+  }
+}
+
+export const electronLogger =
+  (packageName: string, parallelConsoleLog: boolean = false) =>
+  (module: string): QuietLogger => {
+    const name = `${packageName}:${module}`
+    nodeConsoleLogger.info(`Initializing logger ${name}`)
+    return new QuietLogger(name, parallelConsoleLog)
+  }
+
 export const logger = (packageName: string): ((arg: string) => Logger) => {
   return consoleLogger(packageName)
 }
 
+export const createQuietLogger = (
```

Comment:
[style] Can we get a doc string on this and others? I'm having trouble at a glance understanding which loggers exist for which purposes, and where the preferred entrant point are. Like `logger` is a `Logger` type and seems to be used in place of `QuietLogger` in some places. Then this `createQuietLogger` seems to wrap the `electronLogger`, but neither precludes someone from just importing the `QuietLogger` class directly and using it. I'd like to see either a narrower implementation with one point of entry, or more documentation explaining what each of these are used for and the standard way to import and use a logger in our code base.

Reply:
Yea I can clean things up/add doc strings.  Eventually we'll only have the QuietLogger code but see my other comment about why I left in the old logger.

Reply:
For posterity I just went ahead and cleaned up all references.